### PR TITLE
feat: Enable DCR Liveblog rendering for all users

### LINKED ISSUE #4655: Liveblogs to 💯%
Roll out liveblogs to 100% of users once we are happy that the bugs have been squashed

### PR DESCRIPTION
**Pull request by AshCorr**
## What does this change?

Rolls out liveblogs to 100% of users!

Closes guardian/dotcom-rendering#4655

### CODE REVIEW

File: article/app/controllers/LiveBlogController.scala [hunk 1]
```diff
@@ -111,15 +110,14 @@ class LiveBlogController(
...
-            val participatingInTest = ActiveExperiments.isParticipating(LiveblogRendering)
             val isRecent = !LiveBlogController.isNotRecent(blog)
             val theme = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).theme
             val design = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).design
             val display = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).display
             val isDeadBlog = LiveBlogController.isDeadBlog(blog)
             val properties =
               Map(
-                "participatingInTest" -> participatingInTest.toString,
+                "participatingInTest" -> "false",
```
Comment (AshCorr) [inline article/app/controllers/LiveBlogController.scala:120]: Can I just remove this parameter or are there dashboards that rely on it?

**Review by mxdvl — APPROVED** [review]
💯

File: article/app/controllers/LiveBlogController.scala (hunk 1 above)
Comment (mxdvl) [reply article/app/controllers/LiveBlogController.scala:120]: 🤷  probably best to remove in a subsequent PR?

**Review by OllysCoding — APPROVED** [review]

**Review by oliverlloyd — APPROVED** [review]
Hearty approval

**Pull request merged**